rowan/src/Rowan-Tools-Core/RwPrjBrowserTool.class.st

```diff
@@ -618,21 +618,28 @@ RwPrjBrowserTool >> classCreationTemplateForSubclassOf: superclassName className
 ]
 
 { #category : 'class browsing' }
-RwPrjBrowserTool >> classCreationTemplateForSubclassOf: superclassName className: className type: classType instanceVariablesString: instanceVariablesString classVariablesString: classVariablesString classInstanceVariablesString: classInstanceVariablesString poolDictionariesString: poolDictionariesString comment: comment category: category [
+RwPrjBrowserTool >> classCreationTemplateForSubclassOf: superclassName className: className type: classType instanceVariablesString: instanceVariablesString classVariablesString: classVariablesString classInstanceVariablesString: classInstanceVariablesString poolDictionariesString: poolDictionariesString 
+comment: comment category: category [
 
 	"classType may be 'normal', 'bytes', or 'variable'"
 
 	| result lfsp |
 	result := String new.
-	result addAll: superclassName.
 	lfsp := Character lf asString tab.
+	comment isEmpty 
```

Comment:
I'm not sure what the requirements of the class definition template are in this context, but it might make for more readable code if you factor out the setting of the comment into a separate statement and remove the parens. 

Then you'd only need one `` comment isEmpty ifFalse: `` condition.